### PR TITLE
Improve unit tests

### PR DESCRIPTION
**Pull request by clementdessoude**
Create a method to have unit tests on Java code snippets

### CODE REVIEW

File: packages/prettier-plugin-java/test/test-utils.js [hunk 1]
```diff
@@ -69,22 +67,22 @@ function testRepositorySample(testFolder, command, args) {
     );
 
     javaSampleFiles.forEach(fileDesc => {
-      it(`prettify ${relative(samplesDir, fileDesc.path)}`, function() {
+      it(`Performs a stable formatting for <${relative(
+        samplesDir,
+        fileDesc.path
+      )}>`, function() {
         this.timeout(5000);
```
Comment (bd82) [inline packages/prettier-plugin-java/test/test-utils.js]: Why 5000ms timeout?

File: packages/prettier-plugin-java/test/unit-test/snippets/names/methodName/test.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const { expect } = require("chai");
+const { formatJavaSnippet } = require("../../../../test-utils");
+
+describe("MethodName", () => {
+  it("can format a MethodName", () => {
+    const snippet = "test";
```
Comment (bd82) [inline packages/prettier-plugin-java/test/unit-test/snippets/names/methodName/test.spec.js]: these 5 lines keep repeating, perhaps extract to a utility?

File: packages/prettier-plugin-java/test/test-utils.js (hunk 1 above)
Comment (clementdessoude) [reply packages/prettier-plugin-java/test/test-utils.js]: I just kept that from what it was before. @Shaolans may have an answer to why it was added in the first place?

Comment (bd82) [reply packages/prettier-plugin-java/test/test-utils.js]: this may be needed for bigger tests, does not seem appropriate for small unit tests

**Pull request merged**